bootstrap.sh

```diff
@@ -0,0 +1,204 @@
+#!/bin/sh
+
+#
+# Bootstrap 0.1.0 for reMarkable (rM2 untested)
+# wget -qO- '<URL_to_bootstrap.sh>' | sh -s [packages]...
+#
+
+bootstrap_wget() {
+  if [ ! -d /home/root/.wget_bin ]
+  then
+      log INFO "Installing a current wget binary with better and current tls support" \
+          "to /home/root/.wget_bin/ for this installation..." \
+          ""
+      # Bootstrap a current version of wget
+      WGET_BINARIES_PATH='http://static.cosmos-ink.net/remarkable/artifacts'
+      WGET_BINARIES_FILENAME='wget-remarkable-pipeline_job245_wget1.20.3.zip'
+      WGET_BINARIES_SHA256='84185a5934e34e25794d439c78dc9f1590e4df12fbf369236f6a8749bf14d67f'
+
+      # Download and compare to hash
+      wget "$WGET_BINARIES_PATH/$WGET_BINARIES_FILENAME" -O "/home/root/$WGET_BINARIES_FILENAME"
+      if ! echo "$WGET_BINARIES_SHA256  /home/root/$WGET_BINARIES_FILENAME" | sha256sum -c -
+      then
+          echo "FATAL: Invalid hash" >&2
+          exit 1
+      fi
+
+      # Ensure to /home/root/.wget_bin exists and is empty
+      if [ -d /home/root/.wget_bin ]
+      then
+          rm -rf /home/root/.wget_bin/*
+      else
+          mkdir /home/root/.wget_bin
+      fi
+      # Unzip to /home/root/.wget_bin and remove downloaded file
+      unzip "/home/root/$WGET_BINARIES_FILENAME" -d /home/root/.wget_bin -q
+      rm "/home/root/$WGET_BINARIES_FILENAME"
+
+      cat > /home/root/.wget_bin/wget <<EOF
+#!/bin/sh
+LD_LIBRARY_PATH="/home/root/.wget_bin/dist" /home/root/.wget_bin/dist/wget \$@
+EOF
+
+      chmod +x /home/root/.wget_bin/wget
+  fi
+
+  # Ensure this binary is used
+  if [ `which wget` != '/home/root/.wget_bin/wget' ]; then
+    PATH="/home/root/.wget_bin:$PATH"
+  fi
+}
+
+
+main() {
+  # Select source for remarkable_entware repo
+  if [ -z "$REMARKABLE_ENTWARE_REPO_AUTHOR" ]; then
+    # Use LinusCDE's fork for the time beeing
+    # TODO: Switch to "Evidlo" as soon as PRs are merged
+    REMARKABLE_ENTWARE_REPO_AUTHOR=LinusCDE
```

Comment:
this should be pointed to toltec-dev/remarkable_entware whenever we can make that repo (imo)

Reply:
Then I'll remove the explicit mention of Evildo from there and we'll decide that later.

Reply:
Adjusted in commit [d3b53182](https://github.com/toltec-dev/toltec/pull/38/commits/d3b531825d8b46a1b1b2ab89926e0f39b3e4c61c).